### PR TITLE
docs:write_readme

### PR DESCRIPTION
**Pull request by Agnieszka26**

### CODE REVIEW

**Comment by Agnieszka26** [comment]
@UlEdy The checks were not successful because my account reached the limit of free build checks

**Review by UlEdy — COMMENTED** [review]
I suggest removing this mark (^) from the version of every dependency. Also in package.json.

**Comment by Agnieszka26** [comment]
Why from package.json? It has automatically added - when i install package it is updated with "^".

**Comment by UlEdy** [comment]
> Why from package.json? It has automatically added - when i install package it is updated with "^".

"^" makes that new version of package is automaticly added without our control - I suggest removing it and manually installing new versions by ourselves if we decide to do it.

File: README.md [hunk 1]
```diff
@@ -1,37 +1,120 @@
...
+```
+2. When you have the project cloned move to client folder...:
+```bash
+cd pokemon_v2/client
+```
+3. ...where install nessesary dependencies:
+```bash
+yarn install
+```
+note: If you have not installed yarn globally you have to - the project requrires yarn.
```
Comment (UlEdy) [inline README.md:72]: maybe it'll be better to get this information (yarn globally) at the beginning

File: README.md [hunk 2]
```diff
@@ -1,37 +1,120 @@
...
+4. Create yourown enviromental variables in .env file on the same level as app folder with keys:
```
Comment (UlEdy) [inline README.md:73]: missing space between "yourown"

File: README.md [hunk 3]
```diff
@@ -1,37 +1,120 @@
...
+```
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+**Congrats! 🚀 Your fronted is ready!**
+Now, lets move to your backend server:
+
+6. In your terminal run
+```bash
+cd ../server
```
Comment (UlEdy) [inline README.md:89]: we have to open server on the same terminal as client? or we should open new termianl for server?

File: README.md [hunk 4]
```diff
@@ -1,37 +1,120 @@
...
+```
+7. install your dependencies via npm
+```bash
+npm install
+```
+8. make your local backend server up and running:
+```
+npm dev
```
Comment (UlEdy) [inline README.md:97]: it should be `npm start`

**Review by UlEdy — COMMENTED** [review]
A few little mistakes

**Comment by Agnieszka26** [comment]
Please merge ;)

**Pull request merged**